### ISSUE TITLE
There is no Pascal's pre-trained model

### ISSUE THREAD

**Issue by makalo** [Engineering-Course/CIHP_PGN#11]
Hi,I know about you load the pre-trained model of pascal-VOC in your thesis.But you just provide the 'CHIP_PGN' model .Could you provide a tensorflow version of Pascal's pre-training model? I don't know if your Pascal model is transferred from caffe. If so, can you provide a conversion method?

**Comment by Engineering-Course** [comment]
We used the converted pre-trained model, deeplab_resnet_init.ckpt, provided by [Deeplab v2](https://github.com/DrSleep/tensorflow-deeplab-resnet)

**Comment by makalo** [comment]
I start training from  the pre-trained model mentioned above on the Pascal-Person-Part dataset, but only get  mIoU around 66.5, which is much lower than 68.4 reported in the paper. I wonder why the gap exists, and could you please give me your well-trained model on this dataset.

**Comment by Engineering-Course** [comment]
You can train the model on PASCAL dataset, cause we have released the code and implementation details. We have the multi-scale and fliping inference to reach the results in the paper.

**Comment by hanqing1** [comment]
@Engineering-Course Hello,I'm training the model onPASCAL-PERSON-PART dataset,could you tell me how many changes I should do in the script concretly?Thank you.

**Comment by hanqing1** [comment]
@makalo Hello,I also train the model on the Pascal-Person-Part dataset,Could you tell mehow many changes you did in the script?Thank you.

**Comment by DuyenPay** [comment]
@makalo any modification in the scripts (except the number of labels set to be 7) in your training on Pascal? I did it myself and the result was poor, the generated mask was even only all background (generated image with mask was all black)
